.github/workflows/rust.yml

```diff
@@ -32,13 +32,15 @@ jobs:
     - uses: actions/checkout@v2
     - name: Check format
       run: cargo fmt -- --check
-    - name: Run Rust tests
-      run: cargo test
+    - name: Run all unit tests
+      run: cargo test --workspace --all
+    - name: Build all targets (libs, binaries, examples, ..)
+      run: cargo build --workspace --all-targets
     - name: Configure environment
       run: |
         export CARGO_TARGET_DIR=`pwd`/target
```

Comment:
Does this fit with building just before?

Reply:
I would say it may builds even more e.g. if u have examples etc.

Reply:
apparently previous build setup did not build josh-proxy test

Reply:
I mean the CARGO_TARGET_DIR variable.
Looking in detail at the build it looks like it isn't even passed to the script...